pkg/operator/workloads/api_workload.go

```diff
@@ -593,6 +601,7 @@ func onnxAPISpec(
 	servingImage := config.Cluster.ImageONNXServe
 	resourceList := kcore.ResourceList{}
 	resourceLimitsList := kcore.ResourceList{}
+	tolerations := k8s.Tolerations()
```

Comment:
nit: this pattern is not used for the other containers in this file (they use `Tolerations:        k8s.Tolerations(),` directly)